tests/qunit.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>QUnit Example</title>
+
+	<link rel="stylesheet" href="qunit/qunit-1.19.0.css">
+
+	<script src="../js/jquery-2.1.4.min.js"></script>
+	<script src="qunit/qunit-1.19.0.js"></script>
+
+	<script src="../fundraising/banner.js"></script>
+	<script src="../fundraising/banner.config.js"></script>
+	<script type="text/javascript">
+		Banner.config.tracking.libUrl = 'qunit/piwik-dummy.js';
```

Comment:
Having looked at the `Banner.tracking` again I actually think the way you made is better. The way you made it originally we would have all tests use the dummy tracker unless some tests specify otherwise. This is better I think as the risk of adding some invalid tracking data when running tests is lower than if the `libUrl` would be needed to be specified in each test file.